app/forms/idp/create.tsx

```diff
@@ -22,6 +22,11 @@ import { pb } from '~/util/path-builder'
 
 import { MetadataSourceField, type IdpCreateFormValues } from './shared'
 
+const host = window.document.location.host.toString()
+const subdomain = host.includes('localhost')
+  ? 'r3.oxide-preview.com'
+  : host.split('.').slice(2).join('.')
```

Comment:
We need to make super sure of this one.

Reply:
Will move it to a util and get some tests in